modules/challenge/src/main/ChallengeMaker.scala

```diff
@@ -51,7 +49,10 @@ final class ChallengeMaker(
       timeControl = timeControl,
       mode = pov.game.mode,
       color = (!pov.color).name,
-      challenger = Challenge.toRegistered(pov.game.variant, timeControl)(challenger),
+      // for anon, we don't know the secret, but this challenge is only serialized to json and sent to a listening bot anyway,
+      // which doesn't use the secret, so we just use an empty string
+      challenger = challenger
+        .fold(Challenge.Challenger.Anonymous(""))(Challenge.toRegistered(pov.game.variant, timeControl)),
```

Comment:
This is kinda hacky but well, comment explains it. Alternative would be to use `Challenger.Open` but that's kinda even more incorrect. And probably isn't realy worth it to make the secret an Option or something?